src/Block/AdminSearchBlockService.php

```diff
@@ -68,6 +68,14 @@ public function execute(BlockContextInterface $blockContext, Response $response
             $blockContext->getSetting('per_page')
         );
 
+        if (false === $pager) {
+            if (null === $response) {
+                $response = new Response();
```

Comment:
$response = $response ?: new Response(); ?

Reply:
Check the line under, he needs to set the status code.

Reply:
Yes, I am not saying that you return it directly